content/docs/command-reference/fetch.md

```diff
@@ -106,6 +107,13 @@ The `dvc remote` used is determined in order, based on
   workspace. This downloads tracked data for the entire commit history of the
   project.
 
+- `--max-size <bytes>` - fetch data files/directories that are each below specified size
```

Comment:
add `Note:` or even brackets to not distract ppl? Before the The size is determined ...

also, how does it deal with directories - does it skip the whole directory if it's bigger than the threshold or files inside? (I'm just curios if it makes sense semantically to skip the whole dir + might make sense clarifying)

does it support human-readable values?

Reply:
> add Note: or even brackets to not distract ppl? Before the The size is determined ...

Adjusted.

> also, how does it deal with directories - does it skip the whole directory if it's bigger than the threshold or files inside? (I'm just curios if it makes sense semantically to skip the whole dir + might make sense clarifying)

Yes, the whole directory is skipped. Hence why it says `data files/directories that are each below specified size`

> does it support human-readable values?

It does not. Neither does the `size` field, so felt unnecessary for now.

Reply:
what if we have a directory with multiple directories inside - some of them > threshold, some <. 

(I'm trying to understand if that would be clear enough for ppl. )

also, was it the case in the previous implementation in Studio?

Reply:
> what if we have a directory with multiple directories inside - some of them > threshold, some <.

It is using output-level size fields, you can't have output insider of output. So this doesn't happen.

> also, was it the case in the previous implementation in Studio?

To not fetch large files. All I really know 🤷

Reply:
> also, was it the case in the previous implementation in Studio?

I mean was it also skipping the whole directory based on threshold or only files inside it?

Reply:
> It is using output-level size fields, you can't have output insider of output. So this doesn't happen.

I understand this. It's important if people who are going to read this understand this as well. It's kinda not obvious. (not sure though it requires too long clarifications, your call).

May be in that Note add something like. "It means that even if some files or subdirectories are smaller inside a DVC-tracked directory, the whole directory is still skipped"

Reply:
Thanks for the suggestion, adjusted.

Def need to switch to a user mindset more.